src/pages/projects/ProjectsTabs.tsx

```diff
@@ -23,26 +23,27 @@ export default function ProjectsTabs({
 
   const Tab = ({ type }: { type: ProjectCategory }) => {
     return (
-      <Link
-        to={`/projects?tab=${type}`}
-        style={{
-          textTransform: 'uppercase',
-          cursor: 'pointer',
-          borderBottom: '2px solid transparent',
-          paddingBottom: 6,
-          fontWeight: 500,
-          ...(type === selectedTab
-            ? {
-                color: colors.text.primary,
-                borderColor: colors.text.primary,
-              }
-            : {
-                color: colors.text.secondary,
-                borderColor: 'transparent',
-              }),
-        }}
-      >
-        {TAB_TYPE_NAMES[type]}
+      <Link href={`/projects?tab=${type}`}>
+        <a
```

Comment:
**question** same here re nested link

Reply:
To provide a different example, the link itself holds no weight to the child container:
For example, this:
```
    return (
      <Link
        href={`/projects?tab=${type}`}
        style={{
          textTransform: 'uppercase',
          cursor: 'pointer',
          borderBottom: '2px solid transparent',
          paddingBottom: 6,
          fontWeight: 500,
          ...(type === selectedTab
            ? {
                color: colors.text.primary,
                borderColor: colors.text.primary,
              }
            : {
                color: colors.text.secondary,
                borderColor: 'transparent',
              }),
        }}
      >
        {TAB_TYPE_NAMES[type]}
      </Link>
    )
```
renders as:
![image](https://user-images.githubusercontent.com/104132113/179426325-bcd6302d-305f-4ef3-8c95-ad7090937379.png)

Reply:
yeah awesome no worries, cheers for example. we good